TidepoolOnboarding/View Modifiers/NavigationBarAppearance.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  NavigationBarAppearance.swift
+//  TidepoolOnboarding
+//
+//  Created by Darin Krauss on 4/8/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+struct NavigationViewWithNavigationBarAppearance<Content: View>: View {
+    private let content: Content
+    private let navigationBarAppearance: NavigationBarAppearance
```

Comment:
This is good to see, as I think there may be a similar request for a Dexcom screen.